SPECS/kernel/config_aarch64

```diff
@@ -2055,6 +2055,7 @@ CONFIG_PCI_IOV=y
 CONFIG_PCI_PRI=y
 CONFIG_PCI_PASID=y
 CONFIG_PCI_LABEL=y
+# CONFIG_PCI_HYPERV is not set
```

Comment:
Do we want to turn these on for aarch64 VM on Azure?

Reply:
These were previously off (They were removed due to an LSG branch issue. See https://github.com/microsoft/CBL-Mariner/commit/21cd007e4230a84e7ef21253293d1f2110ca872a). But this is a good callout for a future PR